classes/submission/PKPSubmissionFileDAO.inc.php

```diff
@@ -466,23 +465,29 @@ function deleteAllRevisionsByReviewRound($reviewRoundId) {
 
 	/**
 	 * Remove a specific file assignment from a review round.
-	 * @param $submissionId int The submission id of
-	 *  the file
+	 * @param $submissionId int The submission id of the file
```

Comment:
you did a correction below adding a ending point. Another one missing here... :P